org.nongnu.Xlog.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.nongnu.Xlog</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <provides>
+    <binary>xlog</binary>
+  </provides>
+  <name>Xlog</name>
+  <summary>Logging program for Amateur Radio Operators</summary>
+  <description>
+    <p>
+      Xlog is a logging program for Amateur Radio Operators which can be used for daily logging and contest. Logs are stored into a text file.
+      You can open, close and create logs as you need them. Xlog supports cabrillo, ADIF, trlog, and EDI and can import twlog, editest and OH1AA logbook files.
+    </p>
+    <p>
+      QSO's are presented in a list. Items in the list can be added, deleted or updated. For each contact, DXCC information is displayed and bearings and distance is calculated, both short and long path.
+      When hamlib is enabled through the menu, you can retrieve frequency, mode and signal-strength from your rig over the serial port.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://xlog.nongnu.org/xlog.jpg</image>
```

Comment:
This sadly looks to be the best screenshot of Xlog that is available online.

Reply:
I got contacted by the Xlog developer who promised to upload a better screenshot next week. :-)